setup.py

```diff
@@ -16,15 +16,15 @@
     install_requires=[
         "beautifulsoup4==4.7.1",
         "editdistance>=0.5.2, <0.6.0",
-        "emmental==0.0.4",
+        "emmental>=0.0.5, <0.1.0",
         "lxml>=4.2.5, <5.0.0",
         "numpy>=1.11, <2.0",
         "pyyaml>=5.1, <6.0",
         "scipy>=1.1.0, <2.0.0",
-        "snorkel>=0.9.2, <0.10.0",
+        "snorkel @ git+https://github.com/lukehsiao/snorkel.git@luke-patch",
```

Comment:
Need to resolve this before merging. There are 3 options, as I see it.

1. Snorkel bumps up dependencies. I don't know how likely this is to happen (https://github.com/snorkel-team/snorkel/issues/1558).
2. We fork snorkel to HazyResearch org as a more "permanent" fork. Not ideal.
3. We downgrade our use of pytorch to be compatible. Also not ideal.

Any other suggestions?

Reply:
I'd suggest a combination of 1 and 3.
Downgrade pytorch 1.1.0 until Snorkel changes their dependency on pytorch to 1.3.1 or higher.